### PR TITLE
 #163380405 update README.md file

### PR DESCRIPTION
**Pull request by Bobjayafam**
#### What does this PR do?
- update the update README.md file to show the current features of the application
#### Description of Task to be completed?
- write the current features of the app
- write instructions on how to set up a local environment
- include license and author name
#### How should this be manually tested?
- clone the repo
- `cd politico`
- view the README.md file
#### Any background context you want to provide?
N/A
#### What are the relevant pivotal tracker stories?
https://www.pivotaltracker.com/story/show/163380405

### CODE REVIEW

**Comment by Bobjayafam** [comment]
@megame24 kindly review. thanks

File: README.md [hunk 1]
```diff
@@ -1,2 +1,27 @@
 # politico
 Politico- A web application that enables citizens give their mandate to politicians running for different government offices while building trust in the process through transparency.
+
+# Features
+* A User can:
+  * create an account
+  * login into the application
```
Comment (megame24) [inline README.md:7]: I thought a user can do more than the actions listed above

Comment (Bobjayafam) [reply README.md:7]: Yes, a user can do more than that but I only included the features I have currently developed a UI for.

Comment (megame24) [reply README.md:7]: Ok, that means you'll be updating the README as the codebase progresses right?

Comment (Bobjayafam) [reply README.md:7]: Yes I will.

**Comment by megame24** [comment]
You can merge

**Review by megame24 — APPROVED** [review]

**Pull request merged**